components/x-gift-article/storybook/with-gift-credits.js

```diff
@@ -12,7 +12,8 @@ exports.args = {
 		title: 'Title Title Title Title'
 	},
 	showMobileShareLinks: true,
-	id: 'base-gift-article-static-id'
+	id: 'base-gift-article-static-id',
+	enterpriseApiBaseUrl: `https://enterprise-sharing-api.ft.com`
```

Comment:
Question: What about this kind of endpoint? Do you have a way to use environment variables?

Reply:
My understanding is that x-dash should be a standalone package, therefore not relying in the environment it is used. So the API URL is passed as parameter in the component `<GiftArticle />`. When we implement it in next-article it will get the API from an env var (if agreed by the storytelling team). Worth saying that it is the same approach used in the current implementation, but the url is empty in tests, so not visible.